File: worker/tests/fixtures/source/typescript/parse_error.ts
/**
 * TypeScript fixture that intentionally contains a syntax error.
 *
 * Expected parse output:
 *   - FILE node with parse_error=True
 *   - No FUNCTION nodes emitted
 */

// Deliberately malformed: missing closing brace on function body
export function brokenFunction(x: number, y: number): number {
  return x + y;
// missing closing brace intentionally omitted

File: worker/tests/fixtures/source/python/parse_error.py
"""Python fixture that intentionally contains a syntax error.

Expected parse output:
  - FILE node with parse_error=True
  - No FUNCTION nodes emitted (extraction halts on error)
"""

# This function definition is deliberately malformed to trigger tree-sitter parse_error.
def broken_function(x, y
    return x + y  # missing closing paren
